src/services/product.service.js

```diff
@@ -44,6 +44,7 @@ const findProducts = async (role, sellerId, size, page) => {
   }
 
   const products = await Products.findAndCountAll({
+    where:{isExpired},
```

Comment:
Please, double check and Make sure you are using this where in right way,  coz it's main function getProducts not Expired Products, so double check